skus/datastore.go

```diff
@@ -796,6 +804,100 @@ func (pg *Postgres) GetOrderCreds(orderID uuid.UUID, isSigned bool) (*[]OrderCre
 	return nil, nil
 }
 
+// GetOrderTimeLimitedV2Creds returns the order credentials for a OrderID
+func (pg *Postgres) GetOrderTimeLimitedV2Creds(orderID uuid.UUID, isSigned bool) (*[]TimeLimitedV2Creds, error) {
+	// each "order item" is a different record
+	orderCreds := []TimeLimitedV2Creds{}
+	timeAwareCreds := []TimeAwareSubIssuedCreds{}
+
+	// get all of the credentials related to the order_id
+	query := `
+		select
+			order_id, item_id
+			metadata->>'validFrom' as valid_from,
+			metadata->>'validTo' as valid_to,
+			issuer_id, blinded_creds, signed_creds, batch_proof, public_key
+		from order_creds
+		where
+			order_id = $1
+			and valid_from is not null
+			and valid_to is not null
+	`
+
+	if isSigned {
+		query += " and signed_creds is not null"
```

Comment:
This seems a bit suspect.

Reply:
Was copy/pasted from GetOrderCreds method directly.  I think so long as there are no named parameters in the appended string we should be okay.  The alternate is to I guess create the entire query string in the if statement on 828.